package.json

```diff
@@ -43,32 +42,34 @@
     "nomnom": "^1.8.1",
     "object-assign": "^4.1.0",
     "on-finished": "^2.3.0",
+    "pouchdb": "newlogic42/dist-pouchdb",
```

Comment:
`pouchdb` build that includes the https://github.com/pouchdb/pouchdb/pull/6793 fix.